### PR TITLE
Rev the supported requirements

### PR DESCRIPTION
**Pull request by keyur32**
* the Excel 2016 reference was in correct removing that
* bumping version number for Windows, as the "ID" parameter is now required

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810241510242923-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4L31EB6G0R%2BPtS0UUw4R0igOh5m7OVBD4ETbqHVX%2Bpg%3D&st=2018-10-24T15%3A06%3A44Z&se=2018-11-24T15%3A11%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/excel-custom-functions-note.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 > [!NOTE]
-> Custom functions are currently available in developer preview. Custom functions are supported in Excel 2016 for Windows (build 10827 or later), Excel 2016 for Mac (build 13.329 or later), and Excel Online. Additionally, you must join the [Office Insider](https://products.office.com/office-insider) program (**Insider** level -- formerly called "Insider Fast") in order to use custom functions. Custom functions are currently disabled across all Office builds unless you are a member of the Office Insider program (**Insider** level).
\ No newline at end of file
+> Custom functions are currently available in developer preview. Custom functions are supported in Office 365 versions, Excel for Windows (Build 1810 or later), Excel for Mac (build 13.329 or later), and Excel Online. Additionally, you must join the [Office Insider](https://products.office.com/office-insider) program (**Insider** level -- formerly called "Insider Fast") in order to use custom functions. Custom functions are currently disabled across all Office builds unless you are a member of the Office Insider program (**Insider** level).
```
Comment (mscharlock) [inline docs/includes/excel-custom-functions-note.md]: What exactly are "Office 365 versions"? I guess I'm not sure how those differ from Mac/Windows/Online?

Comment (keyur32) [reply docs/includes/excel-custom-functions-note.md]: Should read "Custom Functions are available to developers on an Office 365 subscription, or through Excel Online.  In addition to having Office 365, you must join the [Office Insider] program, and have the following builds. On Excel for Windows (Build 1810 or later), and Excel for Mac (build 13.329 or later)"

Comment (keyur32) [reply docs/includes/excel-custom-functions-note.md]: Can you draft up that please? With the right way to write it? thanks (I think the metapoint is here Office 2016 is not correct and could be misleading)

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810312021210361-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zAh%2B83ZNO5sqwMmr%2BU%2FaSht19za2y3Ynj4T0Fgty4RQ%3D&st=2018-10-31T20%3A17%3A40Z&se=2018-12-01T20%3A22%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/includes/file-tutorial-excel-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/file-tutorial-excel-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810312029099222-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LH2BSS8Y31ofWb3D5Ty8X4brffdDiViz9A%2BWRdRTIL8%3D&st=2018-10-31T20%3A25%3A24Z&se=2018-12-01T20%3A30%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/includes/file-tutorial-excel-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/file-tutorial-excel-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810312107516252-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zTMchCngLjdLCSd9ixLazkpZlirmVJA3S5hTBpGh1UM%3D&st=2018-10-31T21%3A04%3A06Z&se=2018-12-01T21%3A09%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/includes/file-tutorial-excel-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/file-tutorial-excel-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810312216452491-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OAB7LNXm1x2AgqutQMQhiEBbVBLLIw%2FfIr7HrY9UGdA%3D&st=2018-10-31T22%3A12%3A58Z&se=2018-12-01T22%3A17%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/includes/file-tutorial-excel-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/file-tutorial-excel-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810312232243697-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=cklJGeBtFiUWwpgPyHwAoHPA%2BDF4y3K%2FI1Dq9BNBeBw%3D&st=2018-10-31T22%3A28%3A37Z&se=2018-12-01T22%3A33%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/includes/file-tutorial-excel-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/file-tutorial-excel-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810312242135981-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=lSGYe2SmwZBV7tJGNye62xsILEf%2FDR2Hmnz2tZcSoDY%3D&st=2018-10-31T22%3A38%3A27Z&se=2018-12-01T22%3A43%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |
[docs/includes/file-tutorial-excel-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/includes/file-tutorial-excel-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810312252400973-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ZMdAfX2npJIkWwcQJu7GIdrQf%2FlzB4CHFS3g8ciU9QM%3D&st=2018-10-31T22%3A48%3A55Z&se=2018-12-01T22%3A53%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/excel-custom-functions-note.md (hunk 1 above)
Comment (kbrandl) [reply docs/includes/excel-custom-functions-note.md]: I spoke with @keyur32 about this earlier today to clarify my understanding of this info, and I've updated the note accordingly. Thanks, @keyur32!

**Comment by mscharlock** [comment]
Super, thanks! I think this dropped off of my list of things to check on

**Pull request merged**